charts/prometheus-elasticsearch-exporter/templates/podmonitor.yaml

```diff
@@ -0,0 +1,48 @@
+{{- if and .Values.podMonitor.enabled .Values.serviceMonitor.enabled }}
+{{- fail "Either .Values.podMonitor.enabled or .Values.serviceMonitor.enabled can be enabled at a time, but not both." }}
+{{- else if .Values.podMonitor.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: {{ template "elasticsearch-exporter.fullname" . }}
+  {{- if .Values.podMonitor.namespace }}
+  namespace: {{ .Values.podMonitor.namespace }}
+  {{- end }}
+  labels:
+    chart: {{ template "elasticsearch-exporter.chart" . }}
+    app: {{ template "elasticsearch-exporter.name" . }}
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+    {{- if .Values.podMonitor.labels }}
+    {{- toYaml .Values.podMonitor.labels | nindent 4 }}
+    {{- end }}
+spec:
+  jobLabel: {{ default .Release.Name .Values.podMonitor.jobLabel }}
```

Comment:
this is to keep parity with the svc monitor. i can remove this property if you feel we should

Reply:
This default is really meaningless. The chart does not set a label on the pod named after the release, hence, it cannot be used for the job label, resulting in the `job` being set by the operator to its default anyway.

As an example, if I name my release `elasticsearch-exporter`, the chart does not set label `elasticsearch-exporter=somevalue` on the pod so that `job` can be set to `somevalue`. Let us not repeat misunderstanding `jobLabel`.

Reply:
fair enough, removed job label from podmonitor. its not a common parameter i saw anywhere either way